modules/optimization/doc/content/modules/optimization/examples/forceInv_pointLoads.md

```diff
@@ -46,21 +46,60 @@ The [Optimize](Optimize.md) executioner block in [executionerBlock] , provides a
+         caption=Main application `OptimizationReporter` and `Reporter` block for point load
+         parameterization shown in [figSetup]
+
 
 The `MultiApps` block is shown in [multiapps].  The optimization executioner utilizes [FullSolveMultiApp](multiapps/FullSolveMultiApp.md) sub-apps which, when the module is linked in, have special [!param](/MultiApps/FullSolveMultiApp/execute_on) flags to allow the optimization executioner to control which sub-app is being solved.  This allows TAO to call whichever sub-app it needs in order to compute the quantity needed.  For instance, if TAO needs to recompute the objective function with a new set of parameters, it will execute the sub-app with the flag `exectute_on=FORWARD`.  The sub-apps being set-up correspond to the type of optimization algorithm being used.  Gradient free algorithms like Nelder-Mead (`taonm`) only require a forward sub-app.  Gradient based optimization algorithms like conjugate gradient (`taobncg`) require a forward and adjoint sub-app.  Hessian based optimization algorithms like Newton linesearch (`taonls`) requires a forward, adjoint and homogeneous forward sub-app.
 
 !listing test/tests/optimizationreporter/point_loads/main.i
           block=MultiApps id=multiapps
           caption=Main application `MultiApps` block for point load parameterization shown in [figSetup]
 
-`Transfers` makes up the next section of the main input file.  TAO interacts with the various sub-apps using the reporter system and the [MultiAppReporterTransfer.md].  The first three `Transfers` in [transfers] communicate data with the forward sub-app.  These `Transfers` are used by TAO to compute the objective. The first transfer `[toForward_measument]` communicates the measurement locations to the forward app object [OptimizationData.md].  These are the locations where the objective function is computed by minimizing the difference between the simulated and experimental data at discrete points, see [!eqref](theory/InvOptTheory.md#eqn:objective_integral).  The second transfer `[toForward]` is used by TAO to control the parameter being optimized.  In this case, `[toForward]` controls the 'value' reporter in a [ConstantVectorPostprocessor.md] on the forward-app which is then consumed by the [ReporterPointSource.md] dirac kernel to apply the point source loading.  The third transfer, `[fromForward]` returns the simulation values at the measurement points from the forward-app to the main-app `[OptimizationReporter]` which computes a new objective function for TAO.
+`Transfers` makes up the next section of the main input file.  TAO interacts
+with the various sub-apps using the reporter system and the
+[MultiAppReporterTransfer.md].  The first three `Transfers` in [transfers]
+communicate data with the forward sub-app.  These `Transfers` are used by TAO to
+compute the objective. The first transfer `[toForward_measument]` communicates
+the measurement locations to the forward app object [OptimizationData.md].
```

Comment:
```suggestion
compute the objective. The first transfer `[toForward_measument]` communicates
the measurement locations to the forward app object [OptimizationData.md].  This transfer can be avoided by placing the [OptimizationData.md] block on the forward solve input file instead of the main optimization input file.
```